doc/minimal-manual.nix

```
@@ -11,5 +11,12 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ mmdoc ];
 
-  builder = writeScript "${name}-builder" "${mmdoc}/bin/mmdoc nixpkgs $src $out";
+  phases = [ "buildPhase" ];
+
+  buildPhase = ''
+    cp -r $src source
+    chmod -R u+w source
+    cp ${import ./doc-support/lib-functions-docs-cm.nix {}}/*.md source/functions/library/
```

imo `pkgs` should be propagated
```suggestion
    cp ${import ./doc-support/lib-functions-docs-cm.nix { inherit pkgs; }}/*.md source/functions/library/
```